docs/vault.md

```diff
@@ -33,3 +33,22 @@ To lookup information about a token:
 To revoke a token:
 
 `./bldr vault.token_revoke:<token>`
+
+## Reading and writing secrets (admin only)
+
+Some commands can be manually run to directly interact with Vault's key-value secrets store:
+
+```
+$ VAULT_ADDR=https://master-server.elifesciences.org:8200 vault kv get secret/builder/apikey/fastly-gcs-logging
+Key                 Value
+---                 -----
+email               fastly@elife-fastly.iam.gserviceaccount.com
+secret_key          -----BEGIN PRIVATE KEY-----
+...
+-----END PRIVATE KEY-----
```

Comment:
Bit of a weird format...

Reply:
`-format=json` helps:
```
{
  "request_id": "e992d678-58b9-845c-c4f7-477dd2f3e5e2",
  "lease_id": "",
  "lease_duration": 2764800,
  "renewable": false,
  "data": {
    "email": "fastly@elife-fastly.iam.gserviceaccount.com",
    "secret_key": "-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n"
  },
  "warnings": null
}
```
I'll switch to that.